libtiledbsoma/src/soma/soma_array.h

```diff
@@ -515,7 +521,7 @@ class SOMAArray : public SOMAObject {
     /**
      * @brief Get the Arrow schema of the array.
      *
-     * @return std::unique_ptr<ArrowSchema> Schema
+     * @return std::unique_ptr<std::unique_ptr<ArrowSchema>> Schema
```

Comment:
Dual unique pointer.

```suggestion
     * @return std::unique_ptr<ArrowSchema> Schema
```

Reply:
Uber-unique 😎

Reply:
Good point. That means it doesn't exist.  Less-than-zero copy.  We should patent that.